tests/TestCase/Http/ResponseTest.php

```diff
@@ -1833,7 +1833,7 @@ public function testCorsParameters($request, $origin, $domains, $methods, $heade
             $response = new Response();
 
             $result = $response->cors($request, $domains, $methods, $headers);
-            $this->assertInstanceOf('Cake\Network\CorsBuilder', $result);
+            $this->assertInstanceOf('Cake\Http\CorsBuilder', $result);
```

Comment:
Can we use ::class to make refacting easier and allow IDE discovery?

Reply:
This whole method is gonna go away once deprecations are removed. It's a temp fix to get the builds passing.